src/components/NcInputField/NcInputField.vue

```diff
@@ -295,6 +295,10 @@ export default {
 		},
 	},
 
+	created() {
+		this.validateLabel()
+	},
```

Comment:
IMO, it's more reliably to use 1 `computed` + 1 `immediate watch` here than 2 manual watches + `created` hook. Less easy to forgot dependencies or created.

But fine, as it works and that it how it was before.

Reply:
I think I don't really understand what you mean. Feel free to adjust the PR accordingly, if you like.

Reply:
@ShGKme you would warn  in the computed?

Reply:
@skjnldsv No, A meant smth like
```js
computed: {
  isValidLabel() {
    return this.label || this.labelOutside
  },
},

watch: {
  isValideLabel: {
    immediate: true,
    handler() {
      if dont
        warn
    }
  }
}
```

Reply:
Nice idea. How about we warn in the `computed` and actually use it in the template? Then we don't need a watcher at all. See me last commit.